src/components/Footer.style.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+
+export const FooterStyled = styled.footer`
+  display: flex;
+  padding: 0 250px;
+  height: 100px;
+  justify-content: space-between;
```

Comment:
Space-between won't work here unfortunately. Try to use `flex: 1` for the left and right links. Also play with top and left margin of the logo to make it pixel perfect ;-)

![image](https://user-images.githubusercontent.com/9722792/77833444-a3024480-713d-11ea-9bde-68691064d0ad.png)